src/ngraph_builder.cc

```diff
@@ -1249,11 +1271,11 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
       TF_RETURN_IF_ERROR(op->input_node(3, &tf_mean));
       TF_RETURN_IF_ERROR(op->input_node(4, &tf_variance));
 
-      auto ng_input = ng_op_map.at(tf_input->name());
-      auto ng_scale = ng_op_map.at(tf_scale->name());
-      auto ng_offset = ng_op_map.at(tf_offset->name());
-      auto ng_mean = ng_op_map.at(tf_mean->name());
-      auto ng_variance = ng_op_map.at(tf_variance->name());
+      auto ng_input = GetNgOp(ng_op_map, tf_input->name());
```

Comment:
All of these GetNgOp will be replaced by TF_RETURN_IF_ERROR(...) once the GetNgOp(...) is modified to return TF Status (or whatever the error object is).